web/src/main/react-dashboard/src/components/DeploymentPatternView.js

```diff
@@ -137,6 +137,27 @@ class DeploymentPatternView extends Component {
                 </table>
               </Subheader>;
             case PENDING:
+                return <Subheader style={{
+                    fontSize: '20px',
+                    backgroundColor: "#cdffba"
+                }}>
+                  <table>
+                    <tbody>
+                    <tr>
+                      <td style={{ padding: 5 }}>
+                        <img
+                            src={require('../new.png')}
```

Comment:
Icon file name "new" doesn't make sense here. Can we rename it as "pending" or something?